packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1690,36 +1690,49 @@ class EditableText extends StatefulWidget {
     required final VoidCallback? onCut,
     required final VoidCallback? onPaste,
     required final VoidCallback? onSelectAll,
+    required final VoidCallback? onLiveTextInput,
   }) {
-    // If the paste button is enabled, don't render anything until the state
-    // of the clipboard is known, since it's used to determine if paste is
-    // shown.
+    final List<ContextMenuButtonItem> resultButtonItem = <ContextMenuButtonItem>[];
+
+    // Config button items with clip board.
     if (onPaste != null && clipboardStatus == ClipboardStatus.unknown) {
```

Comment:
It's strange that this is now an empty `if` block.  Why did you refactor it to declare `resultButtonItem` instead of returning the list directly like it was?  If you do keep `resultButtonItem`, maybe just reverse the logic in this `if` so that there is no `else` case.